CHANGELOG.md

```diff
@@ -37,14 +46,14 @@ include updated documentation and files outlined by the
 
 Repository structure changes include:
 
-* Migrating `pymods` directory to `swath_projector`.
-* Migrating `swotrepr.py` to `swath_projector/adapter.py`.
-* Addition of `swath_projector/main.py`.
+- Migrating `pymods` directory to `swath_projector`.
+- Migrating `swotrepr.py` to `swath_projector/adapter.py`.
+- Addition of `swath_projector/main.py`.
 
 For more information on internal releases prior to NASA open-source approval,
 see legacy-CHANGELOG.md.
 
-[v1.1.1]:(https://github.com/nasa/harmony-swath-projector/releases/tag/1.1.0)
-[v1.1.0]:(https://github.com/nasa/harmony-swath-projector/releases/tag/1.0.1)
-[v1.0.1]:(https://github.com/nasa/harmony-swath-projector/releases/tag/1.0.1)
-[v1.0.0]:(https://github.com/nasa/harmony-swath-projector/releases/tag/1.0.0)
+[v1.1.1]: (https://github.com/nasa/harmony-swath-projector/releases/tag/1.1.0)
+[v1.1.0]: (https://github.com/nasa/harmony-swath-projector/releases/tag/1.0.1)
```

Comment:
This should be:
`[v1.1.0]: (https://github.com/nasa/harmony-swath-projector/releases/tag/1.1.0)`

Reply:
In addition to that change, I'm fixing the other line to be:
`[v1.1.1]: (https://github.com/nasa/harmony-swath-projector/releases/tag/1.1.1)`

Reply:
Fixed in commit [735894d](https://github.com/nasa/harmony-swath-projector/pull/19/commits/735894d1f7c3771c9aa53047e64669af8db5f877)